src/EnergyPlus/SetPointManager.cc

```diff
@@ -8391,7 +8391,7 @@ void DefineIdealCondEntSetPointManager::SetupMeteredVarsForSetPt(EnergyPlusData
     unitsForVar.allocate(NumVariables);
 
     for (int varN = 1; varN <= NumVariables; ++varN) {
-        ResourceTypes.insert(std::pair<int, Constant::ResourceType>(varN, Constant::ResourceType::None));
```

Comment:
This is a bad use of `std::map` because the key (`varN`) is a dense integer set.

Reply:
@dareumnam another item to do in the next round, or if not, then explain why it's not being changed.

Reply:
Oops, I missed this one. I'll address this on the follow-up PR.